cves/kernel/CVE-2021-3609.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A flaw in the business contiunity management protocol in the CAN subsystem of linux
```

Comment:
Probably worth clarifying what BCM and the CAN subsystem are. 

also, change "contiunity" to "continuity "